back/app/services/survey_results_generator_service.rb

```diff
@@ -49,7 +49,11 @@ def visit_linear_scale(field)
     values = inputs
       .select("custom_field_values->'#{field.key}' as value")
       .where("custom_field_values->'#{field.key}' IS NOT NULL")
-    distribution = Idea.select(:value).from(values).group(:value).order(Arel.sql('COUNT(value) DESC')).count.to_a
+    distribution_from_db = Idea.select(:value).from(values).group(:value).order(value: :desc).count.to_h
+    distribution = []
+    (1..field.maximum).reverse_each do |value|
```

Comment:
As city admins love customizability so much, I wouldn't be surprised if at some point a request comes in to be able to have different minimum values than "1".